datameta/api/openapi.yaml

```
@@ -736,6 +768,15 @@ components:
       items:
         $ref: "#/components/schemas/SubmissionResponse"
 
+    GroupNameChange:
+      type: object
+      properties:
+        newGroupName:
```

```suggestion
        name:
```
Just use the attribute name here

---

Done